app/models/dialog_field_serializer.rb

```diff
@@ -17,7 +17,7 @@ def serialize(dialog_field, all_attributes = false)
     }
 
     if dialog_field.try(:force_multi_value) && dialog_field.default_value.present?
-      extra_attributes["default_value"] = dialog_field.default_value.split(",")
+      extra_attributes["default_value"] = dialog_field.default_value
```

Comment:
I'm a bit confused...seems like we just need to revert https://github.com/ManageIQ/manageiq/pull/16555 entirely since it is just passing back the `default_value` without any changes to it?

Reply:
@jntullo I'll probably update the PR again, as a result of the review https://github.com/ManageIQ/manageiq-ui-classic/pull/3094#issuecomment-355000552

I need to discuss it a bit more, as I'm a bit confused as well.
Right now, after closing the https://github.com/ManageIQ/manageiq-ui-classic/pull/3094 , I'd change the line to:

```ruby
extra_attributes["default_value"] = JSON.parse(dialog_field.default_value).join(',')
```

(taking a JSON array -- `"[\"a\", \"b\", \"c\"]" ` and converting it to a comma split string -- `"a,b,c"`, which also doesn't seem to be correct according to discussion in https://github.com/ManageIQ/manageiq-ui-classic/pull/3094)

Marking this as WIP for now.